c2corg_ui/static/js/card.js

```diff
@@ -223,7 +355,12 @@ app.CardController.prototype.getGlobalRatings = function() {
  * @return {Object} ratings
  */
 app.CardController.prototype.getFullRatings = function() {
-  var doc = this.doc;
+  var doc = {};
+  if (this.type == 'feeds') {
+    doc = this.doc['document'];
+  } else {
+    doc = this.doc;
+  }
```

Comment:
```
var doc = this.type == 'feeds' ? this.doc['document'] : this.doc;
```

But why not adding a function that would figure out once for all where the document data are stored, and call this function wherever you make those "type" tests?
```
app.CardController.prototype.getDoc_ = function() {
  return this.type == 'feeds' ? this.doc['document'] : this.doc;
};
```